testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -709,6 +716,7 @@ public static class Builder<SELF extends Builder<?>>
         private List<EventListener> eventListeners = ImmutableList.of();
         private ImmutableList.Builder<AutoCloseable> extraCloseables = ImmutableList.builder();
         private TestingTrinoClientFactory testingTrinoClientFactory = TestingTrinoClient::new;
+        private Optional<String> encodingId = Optional.empty();
```

Comment:
spoolingEncoding?

Reply:
not consistent with the existing code

Reply:
Codec is not a right word (as this denotes COmpression / DECompression)
SerDe is not a right word too (SERialization / DEserialization)

Encoding is both codec and serde hence the name

Reply:
Codec is coding/decoding :)